### PR TITLE
Fix broken link in docs

### PR DESCRIPTION
**Pull request by 3nprob**

### CODE REVIEW

File: docs/code_contribution_guidelines.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ A quick summary of test practices follows:
   or RPC's will need to be accompanied by integration tests which use the
   [`networkHarness`framework](https://github.com/lightningnetwork/lnd/blob/master/lntest/harness.go)
   contained within `lnd`. For example integration tests, see
-  [`lnd_test.go`](https://github.com/lightningnetwork/lnd/blob/master/lnd_test.go#L181). 
+  [`lnd_test.go`](https://github.com/lightningnetwork/lnd/blob/master/lntest/itest/lnd_test.go#L163).
```
Comment (bhandras) [inline docs/code_contribution_guidelines.md:157]: I think it's safe to remove the line part.

**Comment by yyforyongyu** [comment]
Thanks for the fix! Please refer to [our contribution guide](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#ideal-git-commit-structure) on how to write git commit messages.

**Review by bhandras — APPROVED** [review]
LGTM, thanks! :+1: 

One more thing is needed, please update `docs/release-notes/release-notes-0.14.0.md` (add this PR to the appropriate section) so that GitHub CI doesn't complain.

**Comment by 3nprob** [comment]
> LGTM, thanks! +1
> 
> One more thing is needed, please update `docs/release-notes/release-notes-0.14.0.md` (add this PR to the appropriate section) so that GitHub CI doesn't complain.

Done!

**Comment by guggero** [comment]
Replaced by https://github.com/lightningnetwork/lnd/pull/6080 to speed up review process. Original commit author was preserved, thank you for your contribution.

**Pull request closed without merging**